modules/gdnative/gdnative.h

```diff
@@ -43,7 +43,8 @@
 class GDNativeScriptData;
 class GDNativeLibrary;
 
-struct NativeLibrary {
+class NativeLibrary {
```

Comment:
This creates warning when compiling with clang

```
In file included from modules/gdnative/register_types.cpp:31:
modules/gdnative/gdnative.h:212:9: warning: class 'NativeLibrary' was previously declared as a struct [-Wmismatched-tags]
        friend class NativeLibrary;
               ^
modules/gdnative/gdnative.h:46:8: note: previous use is here
struct NativeLibrary {
       ^
modules/gdnative/gdnative.h:212:9: note: did you mean struct here?
        friend class NativeLibrary;
               ^~~~~
               struct
```
So we can either declare them as `class` or write `friend struct NativeLibrary`